docs/pages/cloud/faq.mdx

```diff
@@ -21,6 +21,10 @@ this number reflects risks in our architecture that we will improve over time.
 
 [Reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
+## If I start with Teleport Cloud, can I move to self-hosted or community, or do I need to start again?
+
+If you also plan to use S3 and DynamoDB, we can provide data for you to import. But you should reach out to us first.
```

Comment:
```suggestion
If you plan to use S3 and DynamoDB as storage backends, we can provide data for you to import. But you should reach out to us first.
```

Reply:
To make this a bit more explicit, I think we should also add a sentence indicating what happens if you plan to use another storage backend.

Reply:
Maybe "If you use a different backend, you will need to start over."?  I don't know if we have any migration path from the Cloud's S3/DynamoDB to other backends.

Reply:
Works for me!